Classes/Domain/Repository/TeaRepository.php

```diff
@@ -21,6 +21,7 @@ class TeaRepository extends Repository
      *
      * @return QueryResultInterface<Tea>
      */
+    /** @phpstan-ignore-next-line missingType.generics */
```

Comment:
This is something that we want to fix (in the Core, in our code, or with changing a type annotation). So we shouldn't have this blocklist line.